test/mappers/second_quantization/test_bksf_mapper.py

```diff
@@ -0,0 +1,157 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test Bravyi-Kitaev Super-Fast Mapper """
+
+import unittest
+from test import QiskitNatureTestCase
+
+import numpy as np
+from qiskit.quantum_info import SparsePauliOp
+
+from qiskit_nature.operators.second_quantization import FermionicOp
+from qiskit_nature.mappers.second_quantization import BravyiKitaevSFMapper
+from qiskit_nature.mappers.second_quantization.bksf import edge_operator_aij, edge_operator_bi
+import qiskit_nature.mappers.second_quantization.bksf as bksf
+
+
+def _sort_simplify(sparse_pauli):
+    sparse_pauli = sparse_pauli.simplify()
+    indices = sparse_pauli.table.argsort()
+    table = sparse_pauli.table[indices]
+    coeffs = sparse_pauli.coeffs[indices]
+    sparse_pauli = SparsePauliOp(table, coeffs)
+    return sparse_pauli
+
+
+class TestBravyiKitaevSFMapper(QiskitNatureTestCase):
+    """Test Bravyi-Kitaev Super-Fast Mapper"""
+
+    def test_bksf_edge_op_bi(self):
+        """Test bksf mapping, edge operator bi"""
+        edge_matrix = np.triu(np.ones((4, 4)))
+        edge_list = np.array(np.nonzero(np.triu(edge_matrix) - np.diag(np.diag(edge_matrix))))
+        qterm_b0 = edge_operator_bi(edge_list, 0)
+        qterm_b1 = edge_operator_bi(edge_list, 1)
+        qterm_b2 = edge_operator_bi(edge_list, 2)
+        qterm_b3 = edge_operator_bi(edge_list, 3)
+
+        ref_qterm_b0 = SparsePauliOp("IIIZZZ")
+        ref_qterm_b1 = SparsePauliOp("IZZIIZ")
+        ref_qterm_b2 = SparsePauliOp("ZIZIZI")
+        ref_qterm_b3 = SparsePauliOp("ZZIZII")
+
+        with self.subTest("Test 1"):
```

Comment:
Can we give a more descriptive name than Test 1? What does it test exactly? Applies to other places too.

Reply:
Done